demisto_sdk/commands/split/jsonsplitter.py

```diff
@@ -124,4 +146,69 @@ def create_module(self):
 
             module_file_path = os.path.join(self.module_dir, file_name)
 
-        write_dict(module_file_path, data=self.module_json_data, indent=4)
+        write_dict(module_file_path, data=self.json_data, indent=DEFAULT_JSON_INDENT)
+
+    def get_auto_output_path(self) -> Tuple[Path, str, str]:
+        """
+        Obtains the output path automatically according to the List name
+        if autocreate_dir == true, otherwise returns the output as is
+        """
+        suffix_by_type = {
+            TypeListData.TEXT: ".txt",
+            TypeListData.HTML: ".html",
+            TypeListData.CSS: ".css",
+            TypeListData.MD: ".md",
+            TypeListData.JSON: ".json",
+        }
+
+        suffix = suffix_by_type.get(self.json_data["type"], ".txt")
+
+        file_name = pascal_case(self.json_data["name"]) + ".json"
+        file_data_name = file_name[: -len(".json")] + "_data" + suffix
+
+        if self.autocreate_dir:
+            pack_name = get_pack_name(self.input)
+
+            if not pack_name:
+                return Path(self.input).parent, file_name, file_data_name
+
+            (lists_dir := Path(PACKS_DIR) / pack_name / LISTS_DIR).mkdir(exist_ok=True)
+
+            # create the specific list dir under the LISTS_DIR if it does not exist
+            # the dir name determine by the file name without the suffix
+            (list_name_dir := lists_dir / Path(file_name).stem).mkdir(exist_ok=True)
```

Comment:
```suggestion
            (list_name_dir := Path(PACKS_DIR, pack_name, LISTS_DIR, Path(file_name).stem).mkdir(parents=True, exist_ok=True)
```

1. Removed the comment since it explains what is done, instead of why
2. Using `parents=True` saves the need to create the two dirs separately.